server/core/src/dto/expenses.rs

```diff
@@ -47,3 +47,36 @@ pub struct NewPredefinedExpenseRequest {
     pub currency_type_id: Id,
     pub recurrence_type_id: Id,
 }
+
+pub type ExpenseResponse = Vec<ExpenseWithTransactions>;
+
+#[derive(Clone, Serialize)]
+pub struct ExpenseWithTransactions {
```

Comment:
Why is there a struct named ExpenseWithTransactions both in api and core? It's a bit strange design wise.

Reply:
Yes, the real one is in the core.